### PR TITLE
Add code snippets for React Native tracker

### PR DESCRIPTION
**Pull request by matus-tomlein**
This PR adds code snippets for setting up tracking in the React Native tracker.

### CODE REVIEW

File: content/tracking/2-mobile_trackers_usage.md [hunk 1]
```diff
@@ -95,6 +128,47 @@ In addition to tracking events from the native code, we also want to track event
...
+Snowplow.subscribeToWebViewEvents(webView);
+```
+
+{{% /tab %}}
+
+{{% tab name="React Native" %}}
+
+The tracker supports Web views created using the [React Native WebView package](https://www.npmjs.com/package/react-native-webview).
+
+You can call pass a callback created using `getWebViewCallback()` as a parameter you in your `WebView` component:
```
Comment (AlexBenny) [inline content/tracking/2-mobile_trackers_usage.md]: Is this right?
```suggestion
You can pass a callback created using `getWebViewCallback()` as a parameter in your `WebView` component:
```

File: content/tracking/2-mobile_trackers_usage.md [hunk 2]
```diff
@@ -44,6 +44,26 @@ TrackerController tracker = Snowplow.createTracker(context,
...
+
+{{% tab name="React Native" %}}
+
+```typescript
+import { createTracker } from '@snowplow/react-native-tracker';
+
+const tracker = createTracker(
+    'appTracker',
+    {
+      endpoint: COLLECTOR_URL,
```
Comment (AlexBenny) [inline content/tracking/2-mobile_trackers_usage.md:57]: trailing comma. It's correct in typescript but I've seen you haven't used it later so don't know if you prefer to be consistent.

File: content/tracking/2-mobile_trackers_usage.md [hunk 3]
```diff
@@ -44,6 +44,26 @@ TrackerController tracker = Snowplow.createTracker(context,
...
+    },
+    {
+        trackerConfig: {
+            appId: 'appTracker',
```
Comment (AlexBenny) [inline content/tracking/2-mobile_trackers_usage.md:61]: item (trailing comma)

**Review by AlexBenny — APPROVED** [review]
LGTM!! just added a couple of minor notes

File: content/tracking/2-mobile_trackers_usage.md (hunk 2 above)
Comment (matus-tomlein) [reply content/tracking/2-mobile_trackers_usage.md:57]: Thanks, added trailing commas in all typescript snippets as I think that is the convention.

**Pull request merged**